[2023-07-17T13:22:03.979+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_dag.data_acquisition manual__2023-07-17T13:21:18.056740+00:00 [queued]>
[2023-07-17T13:22:03.989+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_dag.data_acquisition manual__2023-07-17T13:21:18.056740+00:00 [queued]>
[2023-07-17T13:22:03.989+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-17T13:22:04.003+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): data_acquisition> on 2023-07-17 13:21:18.056740+00:00
[2023-07-17T13:22:04.008+0000] {standard_task_runner.py:57} INFO - Started process 217 to run task
[2023-07-17T13:22:04.012+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'data_acquisition', 'manual__2023-07-17T13:21:18.056740+00:00', '--job-id', '690', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmp3jnrv1hw']
[2023-07-17T13:22:04.015+0000] {standard_task_runner.py:85} INFO - Job 690: Subtask data_acquisition
[2023-07-17T13:22:04.061+0000] {task_command.py:410} INFO - Running <TaskInstance: main_dag.data_acquisition manual__2023-07-17T13:21:18.056740+00:00 [running]> on host 46d66aa08365
[2023-07-17T13:22:04.158+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main_dag' AIRFLOW_CTX_TASK_ID='data_acquisition' AIRFLOW_CTX_EXECUTION_DATE='2023-07-17T13:21:18.056740+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-17T13:21:18.056740+00:00'
[2023-07-17T13:23:05.518+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/wikipedia/wikipedia.py:389 GuessedAtParserWarning: No parser was explicitly specified, so I'm using the best available HTML parser for this system ("lxml"). This usually isn't a problem, but if you run this code on another system, or in a different virtual environment, it may use a different parser and behave differently.

The code that caused this warning is on line 389 of the file /home/***/.local/lib/python3.7/site-packages/wikipedia/wikipedia.py. To get rid of this warning, pass the additional argument 'features="lxml"' to the BeautifulSoup constructor.
[2023-07-17T13:23:05.541+0000] {logging_mixin.py:150} INFO - Could not load Casp
[2023-07-17T13:23:36.331+0000] {logging_mixin.py:150} INFO - Could not load Computer Vision
[2023-07-17T13:23:53.692+0000] {logging_mixin.py:150} INFO - Could not load Dbscan
[2023-07-17T13:23:58.754+0000] {logging_mixin.py:150} INFO - Could not load Data Mining
[2023-07-17T13:25:25.387+0000] {logging_mixin.py:150} INFO - Could not load Github
[2023-07-17T13:25:33.885+0000] {logging_mixin.py:150} INFO - Could not load Google
[2023-07-17T13:26:07.882+0000] {logging_mixin.py:150} INFO - Could not load Identity Matrix
[2023-07-17T13:27:14.853+0000] {logging_mixin.py:150} INFO - Could not load Machine Learning
[2023-07-17T13:27:43.730+0000] {logging_mixin.py:150} INFO - Could not load Multilayer Perceptron
[2023-07-17T13:27:48.440+0000] {logging_mixin.py:150} INFO - Could not load Np-Hard
[2023-07-17T13:28:35.646+0000] {logging_mixin.py:150} INFO - Could not load Pmc (Identifier)
[2023-07-17T13:28:41.542+0000] {logging_mixin.py:150} INFO - Could not load Petar Veličković
[2023-07-17T13:28:43.858+0000] {logging_mixin.py:150} INFO - Could not load Physics
[2023-07-17T13:28:46.507+0000] {logging_mixin.py:150} INFO - Could not load Pixel
[2023-07-17T13:29:26.382+0000] {logging_mixin.py:150} INFO - Could not load Relu
[2023-07-17T13:30:12.884+0000] {logging_mixin.py:150} INFO - Could not load Social Graph
[2023-07-17T13:30:44.540+0000] {logging_mixin.py:150} INFO - Could not load Tensorflow
[2023-07-17T13:30:45.997+0000] {logging_mixin.py:150} INFO - Could not load Thomas Kipf
[2023-07-17T13:31:07.183+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-07-17T13:31:07.197+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=data_acquisition, execution_date=20230717T132118, start_date=20230717T132203, end_date=20230717T133107
[2023-07-17T13:31:07.252+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-17T13:31:07.285+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
